crates/kellnr/src/main.rs

```diff
@@ -207,8 +207,14 @@ async fn init_cratesio_prefetch_thread(
     for _ in 0..num_threads {
         let recv2 = recv.clone();
         let db2 = db.clone();
+
+        static UPDATE_CACHE_TIMEOUT_SECS: u64 = 60 * 30; // 30 min cache timeout
+
+        let cache = Cache::builder()
+            .time_to_live(Duration::from_secs(UPDATE_CACHE_TIMEOUT_SECS))
+            .build();
         tokio::spawn(async move {
-            cratesio_prefetch_thread(db2, recv2).await;
+            cratesio_prefetch_thread(db2, cache.clone(), recv2).await;
         });
```

Comment:
Don't really like this, and like I mentioned can move the init code to the cratesio_prefetch module.